packages/cascara/src/structures/BaristaStructure/BaristaStructureDetail.js

```diff
@@ -2,26 +2,30 @@ import React from 'react';
 import pt from 'prop-types';
 import styles from './BaristaStructure.module.scss';
 import classNames from 'classnames/bind';
+import { Role } from 'reakit/Role';
 
 import Loader from '../../private/Loader';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  as: pt.string,
   children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
   header: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
   isLoading: pt.bool,
 };
 
 const BaristaStructureDetail = ({
+  as,
   children,
   header,
   isLoading = false,
   ...rest
 }) => {
   const { className, ...props } = rest;
   return (
-    <div
+    <Role
       {...props}
```

Comment:
Should be ...rest instead of ...props once the other changes are done.

Reply:
This won't be applied since this component is not exported from Cascara